main.tf

```diff
@@ -25,15 +25,36 @@ resource "aws_wafv2_web_acl" "main" {
       name     = lookup(rule.value, "name")
       priority = lookup(rule.value, "priority")
 
-      override_action {
-        dynamic "none" {
-          for_each = length(lookup(rule.value, "override_action", {})) == 0 || lookup(rule.value, "override_action", {}) == "none" ? [1] : []
-          content {}
+      dynamic "override_action" {
```

Comment:
This is changing the behaviour and forces override_action to be specified whereas previously it was defaulting to `none`. Core example requires updating. We also need to expand that examples to see the usage of `geo_match_statement` and potentially `ip_set_reference_statement`